### PR TITLE
Price mutations and relations

### PR DESCRIPTION
**Pull request by manzoorwanijk**
Completes #1939 

## Checklist

* [ ] I have read the documentation relating to systems affected by this pull request, see https://github.com/eventespresso/event-espresso-core/tree/master/docs
* [ ] User input is adequately validated and sanitized
* [ ] all publicly displayed strings are internationalized (usually using `esc_html__()`, see https://codex.wordpress.org/I18n_for_WordPress_Developers)
* [ ] My code is tested.
* [ ] My code follows the Event Espresso code style.
* [ ] My code has proper inline documentation.
* [ ] My code accounts for when the site is in Maintenance Mode (MM2 especially disallows usage of models)

### CODE REVIEW

File: assets/prototype/domain/eventEditor/containers/mutations/tickets.js [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
 	mutation createTicket($input: CreateTicketInput!) {
 		createTicket(input: $input) {
 			ticket {
-				id
-				ticketId
-				description
-				name
-				price
+				...ticketAttributes
+				...ticketPricesAttribute # fetch default prices when a ticket is created.
```
Comment (nicolad) [inline assets/prototype/domain/eventEditor/containers/mutations/tickets.js]: Neat :+1:

File: assets/prototype/domain/eventEditor/containers/mutations/useCreatePriceMutation.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+import useToaster from '../../../../infrastructure/services/toaster/useToaster';
+import useRelations from '../../../../infrastructure/services/relations/useRelations';
+
+/**
+ * @todo replace `tickets` with `useTicketsIn()`
+ */
+const useCreatePriceMutation = ({ tickets, ticketId }) => {
+	const toaster = useToaster();
+	const { updateRelations } = useRelations();
+	const id = 0;
```
Comment (nicolad) [inline assets/prototype/domain/eventEditor/containers/mutations/useCreatePriceMutation.js]: maybe set to `ticketId = 0` instead ?

Comment (manzoorwanijk) [reply assets/prototype/domain/eventEditor/containers/mutations/useCreatePriceMutation.js]: yeah, better

Comment (manzoorwanijk) [reply assets/prototype/domain/eventEditor/containers/mutations/useCreatePriceMutation.js]: done :)

**Pull request merged**